applications/FluidDynamicsBiomedicalApplication/custom_utilities/wss_statistics_utilities.cpp

```diff
@@ -98,32 +98,34 @@ void WssStatisticsUtilities::CalculateWSS(
 
 void WssStatisticsUtilities::CalculateOSI(ModelPart &rModelPart)
 {
-    KRATOS_ERROR_IF_NOT(rModelPart.GetProcessInfo().Has(STEP)) << "'STEP' is not present in '" << rModelPart.FullName() << "' ProcessInfo container." << std::endl;
-    const unsigned int step = rModelPart.GetProcessInfo()[STEP];
-    block_for_each(rModelPart.Nodes(), [&step](NodeType& rNode){
+    constexpr double abs_tol = std::numeric_limits<double>::epsilon();
+    KRATOS_ERROR_IF_NOT(rModelPart.GetProcessInfo().Has(TIME)) << "'TIME' is not present in '" << rModelPart.FullName() << "' ProcessInfo container." << std::endl;
+    KRATOS_ERROR_IF_NOT(rModelPart.GetProcessInfo().Has(DELTA_TIME)) << "'DELTA_TIME' is not present in '" << rModelPart.FullName() << "' ProcessInfo container." << std::endl;
+    const double time = rModelPart.GetProcessInfo()[TIME];
+    const double delta_time = rModelPart.GetProcessInfo()[DELTA_TIME];
+    block_for_each(rModelPart.Nodes(), [&time, &delta_time](NodeType& rNode){

```

Comment:
```suggestion
    block_for_each(rModelPart.Nodes(), [&time, &delta_time, &abs_tol](NodeType& rNode){

```

but it will lose its `constexpr` attribute inside the lambda